README.md

```diff
@@ -25,6 +25,18 @@ npm install -g @akivajgordon/npg
 npg /path/to/my-new-proj
 ```
 
+To initialize the project with Express, add the `--express` option:
+
+```sh
+npx @akivajgordon/npg /path/to/my-new-proj --express
+```
+
+...or
+
+```sh
+npg /path/to/my-new-proj --express
+```
+
```

Comment:
I think this is a little much and repeats a lot of what was there before. Maybe just a short sentence about passing the `--express` flag would be enough. Eventually I'm just going to turn it into a proper "usage" section anyway.

Reply:
I replaced it with just a `[--express] in both original lines

Reply:
Nah, that's not intuitive. That takes too much energy to understand and even then wouldn't be obvious what that's all about.

Reply:
Third time 🍦?